app/javascript/mastodon/features/ui/components/media_modal.js

```diff
@@ -76,9 +76,9 @@ export default class MediaModal extends ImmutablePureComponent {
       const height = image.getIn(['meta', 'original', 'height']) || null;
 
       if (image.get('type') === 'image') {
-        return <ImageLoader previewSrc={image.get('preview_url')} src={image.get('url')} width={width} height={height} alt={image.get('description')} key={image.get('preview_url')} />;
+        return <ImageLoader previewSrc={image.get('preview_url')} src={image.get('url')} width={width} height={height} alt={image.get('description')} title={image.get('description')} key={image.get('preview_url')} />;
```

Comment:
This is useless, I am not sure if the ImageLoader component expects a title prop.